tcms/rpc/tests/test_testexecution.py

```diff
@@ -296,7 +296,7 @@ def test_update_with_non_existing_build(self):
             self.rpc_client.TestExecution.update(self.case_run_1.pk, {"build": 1111111})
 
     def test_update_with_non_existing_assignee(self):
-        with self.assertRaisesRegex(XmlRPCFault, 'Select a valid choice'):
+        with self.assertRaisesRegex(XmlRPCFault, 'Unknown user_id'):
             self.rpc_client.TestExecution.update(self.case_run_1.pk, {"assignee": 1111111})
```

Comment:
You also need tests for non-existing username/email, same thing for existing username/email/id because all of the 3 are possible values.